packages/core/src/agent/durable/preparation.ts

```
@@ -15,6 +15,7 @@ import { deepMerge } from '../../utils';
 import type { Workspace } from '../../workspace';
 import type { Agent } from '../agent';
 import type { AgentExecutionOptions, DelegationConfig } from '../agent.types';
+import { TripWire } from '../trip-wire';
```

ESLint

[error] `../trip-wire` import should occur after import of `../signals`

(import/order)